seeking/module.py

```diff
@@ -29,15 +29,26 @@ async def seeking(self, ctx):
         if ctx.invoked_subcommand is not None:
             return
 
-        embed = utils.discord.create_embed(
-            author=ctx.author,
-            title=_(ctx, "Seeking"),
-        )
         items = SeekingDB.get_all(ctx.guild.id, ctx.channel.id)
 
-        if items:
-            template = "#{id} | {name}, {timestamp}"
-            for item in items:
+        if not items:
+            embed = utils.discord.create_embed(
+                author=ctx.author,
+                title=_(ctx, "Seeking"),
+            )
+            embed.add_field(name="\u200b", value=_(ctx, "No items found"))
+            await ctx.send(embed=embed)
+            return
```

Comment:
Hmm, we usually reply with plain message, embeds are used for actual data. Can you do
```py
if not items:
    await ctx.reply(_(ctx, "No items found."))
    return
```
?

Reply:
Changed.